packages/fxa-content-server/app/styles/tailwind/tooltips.css

```diff
@@ -9,3 +9,55 @@
 .tooltip-container {
   @apply relative;
 }
+
+.input-balloon {
+  @apply absolute text-start w-[calc(100%+.5rem)] -left-1 desktop:w-64 desktop:left-auto px-5 py-4 bg-white rounded-sm top-[calc(100%+.25rem)] desktop:-top-2 border border-grey-100 shadow-sm transition-opacity z-10;
+
+  &::before {
+    @apply absolute w-3 h-3 rotate-45 bg-white border-grey-100 border-l border-t top-[calc(-0.375rem-1px)] content-[''];
+  }
+}
+
+@screen desktop {
+  .input-balloon {
+    @apply before:top-5;
+
+    [dir='ltr'] & {
+      @apply left-[calc(100%+1rem)];
```

Comment:
Kind of torn on writing all of the next styles with a `before:whatever` instead of `&::before` but I feel like sometimes it's nice to separate them so it's not one long line containing everything.